migration/migrator/migrations/system/20200518135745_create_user_data_dir.py

```diff
@@ -0,0 +1,39 @@
+import os
+import shutil
+
+"""Migration for the Submitty system."""
+
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+
+    user_data_path = os.path.join(config.submitty['submitty_data_dir'], 'user_data')
+    user = config.submitty_users['php_user']
+    script_path = os.path.join(config.submitty['submitty_install_dir'], '.setup', 'bin', 'setup_sample_user_data.py')
+
+    # Generate user_data directory
+    if not os.path.isdir(user_data_path):
+        os.system('mkdir ' + user_data_path)
+        os.system('chmod 770 ' + user_data_path)
```

Comment:
Use the `os` functions of `mkdir` and `chmod` instead of `os.system`.

Reply:
Done